rna/admin.py

```diff
@@ -96,6 +96,17 @@ def copy_releases(self, request, queryset):
         else:
             self.message_user(request, 'Copied %s Releases' % release_count)
 
+    def set_to_public(self, request, queryset):
+        """ Set one or several releases to public """
+        release_count = 0
+        for release in queryset:
+            release_count += 1
+            release.is_public = True
+            release.save()
+        if release_count == 1:
+            self.message_user(request, 'Release set to public')
+        else:
+            self.message_user(request, '%s releases set to public' % release_count)
```

Comment:
You can simplify this by using [pluralize](https://docs.djangoproject.com/en/1.6/ref/templates/builtins/#pluralize) if you really want that message, but I'd argue that the message isn't really necessary anyway, since change from the red `-` to the green check is pretty visible. This entire function could just be `queryset.update(is_public=True)`.

Even simpler, you could avoid using a custom admin action at all by adding [list_editable](https://docs.djangoproject.com/en/1.6/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_editable) = ['is_public'] to the class.